plugin/src/main/java/org/opensearch/ml/rest/RestMLUploadModelMetaAction.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.rest;
+
+import static org.opensearch.common.xcontent.XContentParserUtils.ensureExpectedToken;
+import static org.opensearch.ml.plugin.MachineLearningPlugin.ML_BASE_URI;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Locale;
+
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.common.xcontent.XContentParser;
+import org.opensearch.ml.common.transport.upload_chunk.MLUploadModelMetaAction;
+import org.opensearch.ml.common.transport.upload_chunk.MLUploadModelMetaInput;
+import org.opensearch.ml.common.transport.upload_chunk.MLUploadModelMetaRequest;
+import org.opensearch.rest.BaseRestHandler;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.rest.action.RestToXContentListener;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableList;
+
+public class RestMLUploadModelMetaAction extends BaseRestHandler {
+    private static final String ML_UPLOAD_MODEL_META_ACTION = "ml_upload_model__meta_action";
```

Comment:
`model__meta` -> is this intentional?

Reply:
No, it was by mistake. Will remove it in the next PR.